openvdb/unittest/TestPointMove.cc

```diff
@@ -1129,6 +1075,48 @@ TestPointMove::testPointData()
         CPPUNIT_ASSERT_EQUAL(short(0), nonZeroGroups2[2]);
         CPPUNIT_ASSERT_EQUAL(short(1), nonZeroGroups2[3]);
     }
+
+    { // larger data set with a cached deformer and group filtering
```

Comment:
As there's no cpp asserts here, perhaps a slightly more verbose comment regarding what this is testing in regards to this PR

Reply:
I made a small adjustment to verify the values were as expected.